app/v2/controllers/ErrorTranslator.scala

```diff
@@ -80,7 +70,7 @@ trait ErrorTranslator {
         PresentationError.notFoundError(s"Departure movement IDs for ${eori.value} were not found")
       case ArrivalsNotFound(eori) =>
         PresentationError.notFoundError(s"Arrival movement IDs for ${eori.value} were not found")
-      case err: UnexpectedError => PresentationError.internalServiceError(cause = err.thr)
+      case err: errors.PersistenceError.UnexpectedError => PresentationError.internalServiceError(cause = err.thr)
```

Comment:
I'm not really happy with this, I'd much rather you went for the locally scoped imports like we've done in the other converters, above.